### PR TITLE
Run client test on build

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [x] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Execute tests for every client project

## Code changes

Add workflow to run typescript tests on pull request.

- **.github/workflows/test.yml:** Run typescript tests on pull request

## Before you submit

```
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    name: Run tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+
+      - name: Set up Node
+        uses: actions/setup-node@9ced9a43a244f3ac94f13bfd896db8c8f30da67a  # v3.0.0
+        with:
+          # cache: 'npm'
```
Comment (mimartin12) [inline .github/workflows/test.yml:33]: What was your thoughts on commenting this out?

Comment (michalchecinski) [reply .github/workflows/test.yml]: Oh it was a leftover after testing :)

**Review by eliykat — CHANGES_REQUESTED** [review]
This PR will run all the tests in the repository, so you can remove the `npm run test` commands in `build-browser.yml` and `build-cli.yml`, otherwise we'll be doubling up. Thanks!

**Review by eliykat — COMMENTED** [review]
@michalchecinski Looks good to me in terms of what commands you're running, but I'd prefer @mimartin12 or another devops person to do the final review so they can review the GH actions side of things.

**Comment by eliykat** [comment]
PS. also looks like the workflow linter is failing.

**Review by mimartin12 — APPROVED** [review]

**Review by eliykat — APPROVED** [review]

**Pull request merged**